app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SessionsController < ApplicationController
+    def login_receive
+        @user = User.new
+    end
+
+    def create
+        @user = User.new(:email => params[:email], :password=> params[:password])
+
+        if @user.valid?
+            user = User.where(email: @user.email).first
+
+            if user.nil?
+                @user.errors.add(:email, message: "This email is not registered")
+                render :login_receive, status: :unprocessable_entity
+            else
+                if user.password == @user.password
+                    session[:user_email] = @user.email
+                    redirect_to dashboard_index_path
+                else
+                    @user.errors.add(:email, message: "Wrong password")
```

Comment:
You should change the symbol `:email` for `:password` as it is signalling that the problem is related to the user password